pkg/operator/operator.go

```diff
@@ -81,6 +82,7 @@ type Operator struct {
 	daemonsetLister appslisterv1.DaemonSetLister
 	infraLister     configlistersv1.InfrastructureLister
 	networkLister   configlistersv1.NetworkLister
+	pdbLister	policylisterv1.PodDisruptionBudgetLister
```

Comment:
You're not passing the lister and this is nil and it's gonna panic when you use it

Reply:
Yep.  Amazing how much stuff has to change just for that.